zerver/tests/test_alert_words.py

```diff
@@ -203,3 +204,40 @@ def test_alert_flags(self):
         self.assertFalse(self.message_does_alert(user_profile_hamlet, "Don't alert on http://t.co/one/ urls"))
         self.assertFalse(self.message_does_alert(user_profile_hamlet, "Don't alert on http://t.co/one urls"))
 
+    def test_update_alert_words(self):
+        # type: () -> None
+        me_email = 'hamlet@zulip.com'
+        user_profile = get_user_profile_by_email(me_email)
+
+        self.login(me_email)
+        result = self.client_put('/json/users/me/alert_words', {'alert_words': ujson.dumps(['ALERT'])})
+
+        content = 'this is an ALERT for you'
```

Comment:
hmm, somehow I expected the initial message to not be a match in this test.  But I guess this is still pretty reasonable coverage.